docs/DeveloperGuide.md

```diff
@@ -659,86 +669,170 @@ Given below are instructions to test the app manually.
+  
+
 
 [↑ Back to top of section](#7-instructions-for-manual-testing)
 
 [↑ Back to table of contents](#table-of-contents)
 
 <div style="page-break-after: always"></div>
 
+## 7.3 Editing a contact:
+1. Editing a `Person`'s details:
+    1. Prerequisites: List all `Persons` using the `list` command. Multiple `Persons` in the list.
+    2. Test case: `edit 1 b/12/10/2003 tg/example p/88923444`<br>
+        Expected: The first `Person` in the list is edited to have the birthday 12/10/2003, the phone number 88923444. Details of the edited `Person` shown in the status message. If any of these fields were 
+        previously empty, they will be filled with the new information. If any of these fields were previously filled, they will be overwritten with the new information. 
+  2. To see if the `Person`'s details are edited, use the `list` command to verify the details of the edited `Person`.
+  3. For a detailed list of fields that can be edited, see [this](https://ay2223s2-cs2103t-w15-1.github.io/tp/UserGuide.html#5-information-fields--prefixes).
 
-## 7.3 Deleting a person:
+[↑ Back to top of section](#7-instructions-for-manual-testing)
+
+[↑ Back to table of contents](#table-of-contents)
+
+
+## 7.3 Deleting a contact:
 
 1. Deleting a `Person` while all persons are being shown
+   1. Prerequisites: List all `Persons` using the `list` command. Multiple `Persons` in the list.
+   1. Test case: `delete 1`<br>
+      Expected: First `Person` is deleted from the list. Details of the deleted `Person` shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `delete 0`<br>
+      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+      Expected: Similar to previous.
 
-    1. Prerequisites: List all `Persons` using the `list` command. Multiple `Persons` in the list.
+[↑ Back to top of section](#7-instructions-for-manual-testing)
 
-    1. Test case: `delete 1`<br>
-       Expected: First `Person` is deleted from the list. Details of the deleted `Person` shown in the status message. Timestamp in the status bar is updated.
+[↑ Back to table of contents](#table-of-contents)
 
-    1. Test case: `delete 0`<br>
-       Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+<div style="page-break-after: always"></div>
 
-    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-       Expected: Similar to previous.
+## 7.4 Adding additional tags to a contact:
```

Comment:
Ditto, person vs contact. Also probably good to keep the spacing between the heading and the content consistent.

e.g. glossary has spacing between heading of letter and content. this section should have spacing as well maybe?